docs_src/microservices/core/data/Ch-CoreData.md

```diff
@@ -1,102 +1,170 @@
+store the sensor data on the edge system (such as in a
+[gateway](../../../general/Definitions.md#gateway)) until the data gets moved "north" and then exported to
+Enterprise and cloud systems.  Core data persists the data in a local database.  [Redis](https://redis.io/) is used by default, but a database abstraction layer allows for other databases to be used.
+
+!!! Note
+        EdgeX has used [MongoDB](https://www.mongodb.com/) in the past. Through the Geneva release, MongoDB is still supported but is considered deprecated.  Alternate (both open source and commercial) implementations have also been provided in the past. 
+
+Other services and systems, both within EdgeX Foundry and
+outside of EdgeX Foundry, access the sensor data
+only through the core data service. Core data
+could also provide a degree of security and protection of the data collected while the data is at the edge.
+
+Core data has a REST API for moving data into and out of the local
+storage. In the future, core data could be expandable to send or access sensor data via other protocols such as MQTT, AMQP, etc. Core data moves data to the application service (and [edge analytcs](../../../general/Definitions.md#edge-analytics)) via ZeroMQ by default. 
+A message bus abstraction allows core data to use an alternate message bus technology, like MQTT, to send data to the application services.  Use of MQTT requires the installation of a broker such as ActiveMQ.
 
 ## Core Data "Streaming"
 
-By default, Core Data does persist all data collected by devices and
-sensors sent to it. However, when the data is too sensitive to be stored
-at the edge, or the need is not present for data at the edge to be used
-by other services locally (e.g. by an analytics microservice), the data
-can be "streamed" through Core Data without persisting it. A
-configuration change to Core Data (persist.data=false) has Core Data
-send data to the Export Service, through message queue, without
-persisting the data locally. This option has the advantage of reducing
-latency through this layer and storage needs at the network edge, but
-the cost is having no historical data to use for operations based on
-changes over time, and only minimal device actuation decisions, based on
-single event data, at the edge.
+By default, core data persists all data collected by devices sent to it. However, when the data is too sensitive to keep
+at the edge, or there is no use for the data at the edge by other local services (e.g. by an analytics micro service), the data
+can be "streamed" through core data without persisting it. A
+configuration change to core data (PersistData=false) has core data
+send data to the application services without
+persisting the data. This option has the advantage of reducing
+latency through this layer and storage needs at the network edge.  But
+the cost is having no historical data to use for analytics that need to look back in time to make a decision.
```

Comment:
This last sentence talks about the trade off. This is a question I get asked a fair amount. I think it would be more beneficial to explicity state that this is an all or nothing persitence flag and that you cannot specify per device which data is persisted.

Reply:
Addressed with a special note to this section.